Makefile

```diff
@@ -34,21 +36,32 @@ DOCKER_BUILDER_IMAGE_FULL_NAME ?= $(DOCKER_REPO_NAME)/$(DOCKER_BUILDER_IMAGE_NAM
 
 get_sha1sum = $(shell cat $1 | sha1sum | cut -d ' ' -f1)
 
+.PHONY: aws_credentials
+aws_credentials:
+	cat ~/.aws/credentials | awk -F ' = ' 'NF && NR>1 {printf("export %s=%s\n", toupper($$1), $$2)}' > $@
```

Comment:
unfortunately, `awk` doesn't produce the required result on Mac, here's what I have in `aws_credentials` file after run:
```
export AWS_ACCESS_KEY_ID    =<ACTUAL AWS_ACCESS_KEY_ID>
export AWS_SECRET_ACCESS_KEY=<ACTUAL AWS_SECRET_ACCESS_KEY>
export AWS_SESSION_TOKEN    =<ACTUAL AWS_SESSION_TOKEN>
export [<ANOTHER PROFILE IN FILE>]=
export AWS_ACCESS_KEY_ID    =<ACTUAL AWS_ACCESS_KEY_ID>
export AWS_SECRET_ACCESS_KEY=<ACTUAL AWS_SECRET_ACCESS_KEY>
export AWS_SESSION_TOKEN    =<ACTUAL AWS_SESSION_TOKEN>
```
and `make cluster-create` fails with export errors. As a sidenote, there's multiple profiles in `.aws/credentials` so to narrow it down to a single profile one can use something like:
```
grep ${AWS_PROFILE} -A 3 ~/.aws/credentials | tail -n3
```

Here's an idea of how we can achieve the same with `make`:
```
AWS_PROFILE ?=
AWS_ACCESS_KEY_ID ?=
AWS_SECRET_ACCESS_KEY ?=
AWS_SESSION_TOKEN ?=

get_aws_credential = $(shell grep $(AWS_PROFILE) -A 3 ~/.aws/credentials | tail -n3 | grep $1 | xargs | cut -d' ' -f3)

.PHONY: aws_credentials
aws_credentials:
	$(eval AWS_ACCESS_KEY_ID := $(if $(AWS_ACCESS_KEY_ID),$(AWS_ACCESS_KEY_ID),$(call get_aws_credential,aws_access_key_id)))
	$(eval AWS_SECRET_ACCESS_KEY := $(if $(AWS_SECRET_ACCESS_KEY),$(AWS_SECRET_ACCESS_KEY),$(call get_aws_credential,aws_secret_access_key)))
	$(eval AWS_SESSION_TOKEN := $(if $(AWS_SESSION_TOKEN),$(AWS_SESSION_TOKEN),$(call get_aws_credential,aws_session_token)))
```

However, I'm getting the following error with or without the changes from this PR:
```
+ run_terraform plan -state tfdata/terraform.tfstate -out tfdata/plan.out
+ local dcos_terraform_aws_version=v0.2.2-20190606103205
./cluster.sh: line 470: AWS_ACCESS_KEY_ID: unbound variable
make: *** [Makefile:55: cluster-create] Error 1
```

Reply:
>  there's multiple profiles in .aws/credentials 

Oh, I definitely missed that, nice catch! Your idea and code look great since it covers all the edge-cases, and it more generic - I verified it works on my end. Thanks for the neat solution! :thumbsup: